src/SplitPackages/Program.cs

```diff
@@ -206,9 +303,69 @@ private IList<PackageItem> ExpandPackages(IList<PackageItem> packagesFromCsv, st
                 }
             }
 
+            foreach (var item in allPackages)
+            {
+                SetNugetIdentityAndVersion(item);
+            }
+
             return allPackages.ToList();
         }
 
+        private static bool LiteralPatternIsDependency(IList<PackageItem> packages, string packageName)
+        {
+            return packages.Any(p => p.Identity.Equals(packageName, StringComparison.OrdinalIgnoreCase));
+        }
+
+        private static IList<PackageItem> ExpandPackagesFromLiteral(
+            IList<PackageItem> packagesFromSource,
+            string packageName,
+            string destinationFolderName)
+        {
+            var packageFromSource = packagesFromSource
```

Comment:
Would there ever be the case where `packageName` doesn't exist in the source? If so, we should probably log if it can't be found.